.github/workflows/galaxy-publish-collection.yml

```diff
@@ -0,0 +1,27 @@
+name: galaxy-publish-collection
+on:
+  push:
+    paths:
+    - 'os_migrate/galaxy.yml'
```

Comment:
-1 here, i don't think we want to automatically publish on every change to galaxy.yml. More generally, do we want to publish via a GH action at all? Would `./toolbox/run make publish` be better?

Reply:
removed